base/applications/rapps/unattended.cpp

```diff
@@ -22,13 +22,12 @@ BOOL UseCmdParameters(LPWSTR lpCmdLine)
         return FALSE;
     }
 
-    // TODO: use DB filenames as names because they're shorter
-    ATL::CSimpleArray<ATL::CStringW> arrNames;
+    ATL::CSimpleArray<ATL::CStringW> PkgNameList;
     if (!StrCmpIW(argv[1], CMD_KEY_INSTALL))
     {
         for (INT i = 2; i < argc; ++i)
```

Comment:
You probably want to abort this loop (maybe in a later pr?) when you encounter another `/` or `-` prefixed option

Reply:
yes, I'd like to improve command-line parsing in next PR😉